.github/workflows/tests-pro-integration.yml

```diff
@@ -81,10 +81,12 @@ env:
   CI_JOB_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}/attempts/${{ github.run_attempt }}
   # report to tinybird if executed on master
   TINYBIRD_PYTEST_ARGS: "${{ github.ref == 'refs/heads/master' && '--report-to-tinybird ' || '' }}"
+  TESTSELECTION_PYTEST_ARGS: "${{ github.event_name == 'pull_request' && '--path-filter=../localstack/target/testselection/test-selection.txt ' || '' }}"
```

Comment:
This is done "natively" now since our path filter is executed first and THEN the split happens. Since both instances should have the same filter and the pytest split should be deterministic, we shouldn't be missing any tests.